_docs/results/duckassistbot.md

```diff
@@ -8,7 +8,9 @@ order: 1450
 
 ## What is DuckAssistBot and what does it do?
 
-DuckAssistBot is a web crawler for [DuckDuckGo](https://duckduckgo.com). DuckAssistBot’s job is to constantly improve our generated answers displayed on the search engine and to offer users the best and most secure search experience possible. The crawler's user agent will appear as `DuckAssistBot/1.1; (+http://duckduckgo.com/duckassistbot.html)`. It respects [WWW:RobotRules](https://metacpan.org/module/WWW::RobotRules) and originates from these IP addresses:
+DuckAssistBot is a specialized web crawler for [DuckDuckGo Search](https://duckduckgo.com) that crawls pages in real-time for answers generated by [DuckAssist](https://duckduckgo.com/duckduckgo-help-pages/results/duckassist/), which prominently cites its sources. This data is not used in any way to train AI models.  
+
+If you’re a publisher, you may opt out of being a potential source for DuckAssist answers by modifying the [robots.txt](https://metacpan.org/module/WWW::RobotRules) file for your domain(s). Once you’ve updated the file to disallow the DuckAssistBot user agent, the change will take effect after 72 hours and DuckAssistBot will stop crawling your site. The crawler’s user agent will appear as DuckAssistBot/1.1; (+http://duckduckgo.com/duckassistbot.html) and originates from the following IP addresses:
```

Comment:
Since this is an external link, you'll want to add `{:target="_blank"}{:rel="noopener noreferrer"}` after the `(<url>)` to open it in a new tab.

Reply:
I think, for readability, you should format the user agent as code by wrapping it in backticks.